sdk/messaging/src/main/java/com/klaviyo/messaging/InAppMessaging.kt

```diff
@@ -0,0 +1,26 @@
+package com.klaviyo.messaging
+
+import android.app.Activity
+import android.view.ViewGroup
+import com.klaviyo.core.Registry
+import java.io.BufferedReader
+
+object InAppMessaging {
+
+    fun triggerInAppMessage(activity: Activity) {
+        val rootView = activity.getRootViewGroup()
+        val webView = KlaviyoWebView()
+        val html = Registry.config.applicationContext
+            .assets
+            .open("IAMTest.html")
```

Comment:
Obviously just is just the initial work to build out this new module, but should this testing logic be merged at all? Seems like something that should be maybe held in another branch - not the feature branch we're building up for this concept

Reply:
Yeah this is definitely a difference between the ios and Android branches for doing this - personally I figured since it was already written there wasn't much reason to remove. Since we will be just continuing to add to this branch I do like how this shows what it looks like to render a page (specifically one that handles events). If you want though I can yank this and add to a new PR - or remove entirely.

Reply:
That's all pretty fair sounding to me. I'm like 50/50, not sure if we should have testing code set into our feature branch unless we're really sure we'll clean out all the files later but totally get why this is here. In the current state of things we have no other way to easily test this stuff in the initial stages and separating the branch out just creates more overhead in our future work as we constantly aim to re-test our changes

Reply:
All that to say, I would not block on this conversation at all!